rubies/ruby/CVE-2015-1855.yml

```diff
@@ -1,7 +1,7 @@
 ---
 engine: ruby
 cve: 2015-1855
-url: https://www.ruby-lang.org/en/news/2015/04/13/ruby-openssl-hostname-matching-vulnerability/
+url: https://www.ruby-lang.org/en/news/2015/04/13/ruby-openssl-hostname-matching-vulnerability
```

Comment:
It appears that `https://www.ruby-lang.org/` will redirect `https://www.ruby-lang.org/en/news/2015/04/13/ruby-openssl-hostname-matching-vulnerability` back to `https://www.ruby-lang.org/en/news/2015/04/13/ruby-openssl-hostname-matching-vulnerability/`. Better to keep using the canonical URL.

Reply:
both work

Reply:
will fix